WeScan/Edit/EditScanViewController.swift

```diff
@@ -31,7 +31,7 @@ final class EditScanViewController: UIViewController {
     }()
     
     lazy private var nextButton: UIBarButtonItem = {
-        let title = NSLocalizedString("wescan.edit.button.next", tableName: nil, bundle: Bundle(for: EditScanViewController.self), value: "Next", comment: "A generic next button")
+        let title = NSLocalizedString("wescan.edit.button.next", tableName: "Localizable", bundle: Bundle(for: EditScanViewController.self), value: "Next", comment: "A generic next button")
```

Comment:
I disagree with this change, localizations are already working perfectly for me, with the current public and develop versions. Is there any reason to change to this format? Can you confirm that you have tested this in both the sample app and a separate host app using CocoaPods or similar?